test/ProgressOnderwijsUtils.Tests/Data/PocoObjectMapperTest.cs

```diff
@@ -414,4 +415,23 @@ public void VerifyNullability()
         PAssert.That(() => message.Contains("PocoObjectMapperTest.NullablityVerifierPoco.AccountNumber contains NULL despite being non-nullable"));
         PAssert.That(() => !message.Contains("+"));
     }
+
+    [Fact]
+    public void VerifyOfSchemaOnly()
+    {
+        var verifyString = "SalesOrderID,DueDate,ShipDate,Status".Split(',');
+        var dbColumns = ParameterizedSqlForRows(1).OfSchemaOnly().Execute(Connection);
+
+        for (var i = 0; i < verifyString.Length; i++) {
+            PAssert.That(() => dbColumns[i].ColumnName == verifyString[i].Trim());
+        }
+    }
+
+    [Fact]
+    public void VerifyAsMaybe()
+    {
+        var faultyScalar = SQL($"select 1/0");
+        Assert.Throws<Exception>(() => faultyScalar.OfScalar<int>().AsMaybe().Execute(Connection).AssertOk());
```

Comment:
```suggestion
        _ = faultyScalar.OfScalar<int>().AsMaybe().Execute(Connection).AssertError();
```

want anders assert je op een exceptie, en... komt die van AssertOk (zoals bedoeld), of van Execute (niet als bedoeld)?

Reply:
Eh, ik doe 't wel in een nawerkje: https://github.com/progressonderwijs/ProgressOnderwijsUtils/pull/1093